src/app/(components)/Menu.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link"
+import "./Navbar.css" 
+
+const Menu = () => {
+    return (
+        <div className="menuContainer">
+            <Link href={"/"}>Collection</Link>
+            <Link href={"/"}>My Wituka</Link>
+            <Link href={"/"}>Cart and stuff</Link>
+        </div>
```

Comment:
You have placed the "My wituka" and the cart within the menu. However, the menu is supposed to contain the entries:

1. Collection
2. New In
3. 1 Tee = 1 tree
4. Outlet

The My Wituka should actually be an image ( you can take the one from the original website ) and the cart should be its own component, used in the Navbar and not in the menu. Does that make sense?